lib/OpenQA/Scheduler.pm

```diff
@@ -133,7 +133,7 @@ sub job_restart {
     return \@res;
 }
 
-dbus_method('job_set_done', [['dict', 'string', 'string']], ['uint32']);
+dbus_method('job_set_done', [['dict', 'string', 'string']], ['string']);
 sub job_set_done {
```

Comment:
as you're touching it - can't you get rid of yet another IPC for this and put it in the model?